recipes/meraculous/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: meraculous
+    version: "2.2.4"
+
+source:
+    fn: Meraculous-v2.2.4.tar.gz    
+    url: https://sourceforge.net/projects/meraculous20/files/Meraculous-v2.2.4.tar.gz
+    md5: 349feb6cb178643a46e4b092c87bad3a
+
+build:
+    skip: True  # [osx]
+    number: 0
+
+requirements:
+    build:
+    - cmake >=2.8
+    - gcc ==4.8.5 # [linux]
+    - libgd >=2.0
+    - boost-cpp ==1.65.0
+    - perl >=5.10
+    - perl-log-log4perl >=1.31
+    - gnuplot >=3.7
+    run:
+    - libgcc # [linux]
+    - libgd >=2.0
+    - boost-cpp ==1.65.0
+    - perl >=5.10
+    - perl-log-log4perl >=1.31
+    - gnuplot >=3.7
+
+about:
+    home: https://jgi.doe.gov/data-and-tools/meraculous/
+    summary: Meraculous is a whole genome assembler for Next Generation Sequencing data, geared for large genomes. It's hybrid k-mer/read-based approach capitalizes on the high accuracy of Illumina sequence by eschewing an explicit error correction step which we argue to be redundant with the assembly process. Meraculous achieves high performance with large datasets by utilizing lightweight data structures and multi-threaded parallelization, allowing to assemble human-sized genomes on a high-cpu cluster in under a day. The process pipeline implements a highly transparent and portable model of job control and monitoring where different assembly stages can be executed and re-executed separately or in unison on a wide variety of architectures.
+    license: Open Source
```

Comment:
can this be more specific?

Reply:
This is now specified in my second commit.